README.md

```diff
@@ -55,10 +55,16 @@
 
```

Comment:
- [ ] Kindly make sure to remove or replace the Microverse Logo from your README file

Reply:
done thanks alot